builder.md

```diff
@@ -3,7 +3,7 @@
 
 A base16 builder is an application that can build syntax highlighting definition files for text editors by using base16 scheme files which contain a collection of colours and base16 template files which contain syntax highlighting rules.
 
-Builders are designed for theme maintainers' ease of use. Theme maintainers should provide built versions of their theme so the end user doesn't need to be aware of the builder.
+Builders are designed for scheme maintainers' ease of use. Scheme maintainers should provide built versions of their scheme so the end user doesn't need to be aware of the builder.
```

Comment:
This is opposite from the previous meaning.

Reply:
What?

**There are no "theme maintainers"**... 'themes" (the old meaning) maintain themselves when they are built.  There are scheme maintainers and template maintainers... but no "theme maintainers"...

Reply:
Sorry, I could have been clearer - it should be "template" rather than "scheme".

Reply:
Changed to "template".